Orthtree/include/CGAL/Orthtree.h

```diff
@@ -172,6 +172,10 @@ class Orthtree
     with all Orthtree functionality, but any performance benefits are
     unlikely to be realized until `refine()` is called.
 
+    \warning the input point range is rearranged by the
```

Comment:
The kd-tree does copy the points right?

Reply:
Yes.

Reply:
Maybe make it even clearer that input are not copied, stressing it is also a difference with the kd-tree.

Reply:
Maybe even in the user manual as it is important.